dotnet/samples/Demos/TelemetryWithAppInsights/Program.cs

```diff
@@ -294,8 +294,14 @@ private sealed class AIServiceSelector : IAIServiceSelector
                             Temperature = 0,
                             ToolCallBehavior = ToolCallBehavior.AutoInvokeKernelFunctions
                         },
-                        GoogleAIGeminiServiceKey => new GeminiPromptExecutionSettings(),
-                        HuggingFaceServiceKey => new HuggingFacePromptExecutionSettings(),
+                        GoogleAIGeminiServiceKey => new GeminiPromptExecutionSettings()
```

Comment:
Is this change needed?

Reply:
If this is not set, it won't have a default and will be null. This makes sure the activity of the Gemini run has at least one execution setting in this sample app.